### PR TITLE
Temp title for PR

### PR DESCRIPTION
**Pull request by neerajcs19**
fix a particular issue

### CODE REVIEW

File: NeerajChaudhary.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Neeraj Chadhary",
+  "skills": ['learning'],
+  "cause": "the community",
```
Comment (Abhishek-kumar09) [inline NeerajChaudhary.json:4]: Atleast, add  github link, if you dont' have other handles.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

File: NeerajChaudhary.json (hunk 1 above)
Comment (neerajcs19) [reply NeerajChaudhary.json:4]: Ok I add my GitHub link as well

**Comment by Abhishek-kumar09** [comment]
Sorry @neerajcs19 your data should lie here: https://github.com/codeforcauseorg/pledge/tree/master/src/data/contributors
Currently you have put your json file in root directory, which is not perceived at all.
Also please add avatar to your profile.
Here is a sample, you could refer: https://github.com/codeforcauseorg/pledge#steps-to-take-the-pledge

**Comment by neerajcs19** [comment]
Sir I had not understood clearly

**Comment by neerajcs19** [comment]
> Sir I had not understood clearly

@Abhishek-kumar09 you should explain more

**Comment by Abhishek-kumar09** [comment]
@neerajcs19 
Currently you have placed your file (NeerajChaudhary.json) in the main directory.
Instead, your file should lie in `src/data/contributors` directory.
If you would open `src/data/contributors`  directory in your code editor, you could see many such files from other contrubutors too. You have to just place your file there.

Also, 
You should follow this template:
You have not added your avatar in your json file, please add in the image field of your json:
```
{
    "name": "Ganga Chaturvedi",
    "skills" : ["JS", "React"],
    "cause" : "empowerment",
    "image" : "Avatar2",
    "github" : "https://github.com/gangachatrvedi",
    "twitter" : "https://twitter.com/gangachatrvedi",
    "linkedin" : "https://www.linkedin.com/in/gangachatrvedi"
}
```

Please let me know if it helped.

File: NeerajChaudhary.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Neeraj Chadhary",
+  "skills": ['C++''Java'],
```
Comment (Abhishek-kumar09) [inline NeerajChaudhary.json:3]: Also, please add a comma between "c++" and "java" :)

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

File: NeerajChaudhary.json (hunk 2 above)
Comment (neerajcs19) [reply NeerajChaudhary.json]: Yes it helped me

**Comment by Abhishek-kumar09** [comment]
nicely done, thanks for your pr, welcome to the community.

**Pull request merged**